include/umf/mempolicy.h

```diff
@@ -0,0 +1,59 @@
+/*
+ *
+ * Copyright (C) 2024 Intel Corporation
+ *
+ * Under the Apache License v2.0 with LLVM Exceptions. See LICENSE.TXT.
+ * SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+ *
+ */
+
+#ifndef UMF_MEMPOLICY_H
+#define UMF_MEMPOLICY_H 1
+
+#include <umf/base.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+typedef struct umf_mempolicy_t *umf_mempolicy_handle_t;
+typedef const struct umf_mempolicy_t *umf_const_mempolicy_handle_t;
+
+typedef enum umf_mempolicy_membind_t {
+    /// Interleave memory from all memory in memspace
+    UMF_MEMPOLICY_INTERLEAVE,
+    /// Bind memory to namespace
+    UMF_MEMPOLICY_BIND,
+    /// Prefer memory from namespace but fallback to other memory if not available
+    UMF_MEMPOLICY_PREFERRED
+} umf_mempolicy_membind_t;
+
+///
+/// @brief Creates a new memory policy
+/// @param Bind memory binding policy
```

Comment:
Bind -> bind (this is what exactly the parameter is called)

Reply:
done